src/sharedHooks/useCurrentUser.ts

```diff
@@ -1,19 +1,19 @@
 import { RealmContext } from "providers/contexts";
 import { useEffect, useState } from "react";
 import Realm from "realm";
-import type { RealmUser } from "realmModels/types";
```

Comment:
Random question to anyone who may know the answer– why do these `realmModels/types` type definitions exist? It seems like we could (and should?) always use the types from the actual models. Maybe a legacy artifact?

Reply:
we have some open questions about typing as we navigate the gradual migration from js and flow to typescript. we probably won't have guidelines or strong opinions about project-level typing decisions until we're closer to eliminating js/flow. For now, we're fairly comfortable keeping most types loose just to make the js=>ts file transitions. 

That said, @frewsxcv , your js=>ts contributions are a good help in that direction of getting to a point where we can make more holistic efforts for type strategy so thank you!